src/lib.rs

```diff
@@ -258,6 +258,15 @@ impl<T: Runtime> Client<T> {
         &self.metadata
     }
 
+    /// Fetch the value under an unhashed storage key
+    pub async fn fetch_unhashed(
+        &self,
+        key: &StorageKey,
+        hash: Option<T::Hash>,
+    ) -> Result<Option<StorageData>, Error> {
```

Comment:
It might make sense to also perform the decoding rather than return just the raw data?